kfs-core/src/main/java/edu/arizona/kfs/pdp/service/impl/PaymentFileValidationServiceImpl.java

```diff
@@ -191,9 +420,109 @@ else if (StringUtils.isNotBlank(paymentAccountDetail.getFinSubObjectCode())) {
         }
 
     }
-
+    
+    protected void removeGroupFromFileTotals(PaymentFileLoad paymentFile, PaymentGroup pg) {
+        for (PaymentDetail pd : pg.getPaymentDetails()) {
+            removeDetailFromFileTotals(paymentFile, pd); 
+        }
+    }
+    
+    protected void removeDetailFromFileTotals(PaymentFileLoad paymentFile, PaymentDetail pd) {
+        paymentFile.setPaymentCount(paymentFile.getPaymentCount() - 1);
+        paymentFile.setPaymentTotalAmount(paymentFile.getPaymentTotalAmount().subtract(pd.getNetPaymentAmount()));
+    }
+    
+    protected boolean processAccountingEdits(PaymentFileLoad paymentFile, PaymentGroup paymentGroup, int groupCount, List<String> warnings) {

```

Comment:
You are right, this arg is not used in this method. I checked the KFS 3.0 code and it is the same, not used by the method.